From d0e827fc241c63f82dbbbb5b5583686195a39e8e Mon Sep 17 00:00:00 2001
From: Liang Chen <cl@rock-chips.com>
Date: Wed, 29 Nov 2017 11:07:06 +0800
Subject: [PATCH] ARM: dts: rk312x: pinctrl pullup UART1 RX pin

UART RX pin need pullup to keep high voltage when bus is idle.

Change-Id: Iaeea6535bfc42d3ffec396798778cf8803cf3e90
Signed-off-by: Liang Chen <cl@rock-chips.com>
---
 arch/arm/boot/dts/rk312x.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk312x.dtsi b/arch/arm/boot/dts/rk312x.dtsi
index 5f900b6912cb..2e927d66f91b 100644
--- a/arch/arm/boot/dts/rk312x.dtsi
+++ b/arch/arm/boot/dts/rk312x.dtsi
@@ -1144,7 +1144,7 @@
 		uart1 {
 			uart1_xfer: uart1-xfer {
 				rockchip,pins = <1 RK_PB1 2 &pcfg_pull_default>,
-						<1 RK_PB2 2 &pcfg_pull_none>;
+						<1 RK_PB2 2 &pcfg_pull_default>;
 			};
 
 			uart1_cts: uart1-cts {
-- 
2.35.3

